### ISSUE TITLE
resnet50 accuracy on imagenet32

### ISSUE THREAD

**Issue by Lyan-ing** [sndnyang/imagenet64x64#1]
你在imagenet32上做的resnet50实验最高只有38%的精度吗，我看谷歌论文里resnet34做到了60%，但是我自己最高只做到34%,想问问有没有训练技巧

**Comment by sndnyang** [comment]
谷歌哪篇论文？  名字叫resnet34, 但结构可能天差地别。

**Comment by Lyan-ing** [comment]
不好意思，现在才看到。是谷歌做蒸馏的一篇论文，并不是专门研究结构的。Large-Scale Generative Data-Free Distillation， arXiv:2012.05578。里面是把resnet34作为教师网络，没有具体介绍网络结构，他们在32像素上的结果很高，我最高只有三十多，我太菜了。
![图片](https://user-images.githubusercontent.com/65017880/158298556-abbe0cd3-7a32-46a8-a7c6-9123858ced51.png)

**Comment by sndnyang** [comment]
建议联系 论文作者嘛， 虽然他的github里没放这论文的代码。
我只是使用 pytorch官方imagenet原版的设置，  你看那个 loss/acc 曲线图， learning rate 衰减之后 走向明显有问题， 只是一个简单的尝试（复现了wideresnet的）， 主要是数据集格式问题， 不是科研项目，所以我也没管那么多。

**Comment by Lyan-ing** [comment]
我也联系作者了，他还没回我，谢谢啦

**Comment by Lyan-ing** [comment]
嗯，他回我了，实际上他们是对网络结构做了调整，应该是把resnet前两次降维去掉，（stride变为1）。让最后的特征图和224输入的特征图大小接近。我后来试了一下，精度上升到了50多（我只把第一个降采样操作改掉，因为两个都改好像没有更好的效果，可能是我的操作有问题）。我把他的邮件转给你，不太确定我的理解是不是有误。你看一下，我们可以讨论

**Comment by sndnyang** [comment]
> 嗯，他回我了，实际上他们是对网络结构做了调整，应该是把resnet前两次降维去掉，（stride变为1）。让最后的特征图和224输入的特征图大小接近。我后来试了一下，精度上升到了50多（我只把第一个降采样操作改掉，因为两个都改好像没有更好的效果，可能是我的操作有问题）。我把他的邮件转给你，不太确定我的理解是不是有误。你看一下，我们可以讨论

那我没猜错，  他们改结构了， 模型结构名字相同， 内部结构差别这些东西可不好玩。

**Comment by Lyan-ing** [comment]
是的，他们参考了一个工作对结构做了调整

**Comment by sndnyang** [comment]
这model resolution论文看起来挺有意义的， 应该有助于 学习如何分析模型， 毕竟我没研究过 模型结构设计、改进这方向， 但这个又非常有用 👍 
感谢转发

**Comment by sndnyang** [comment]
https://arxiv.org/pdf/1909.03205.pdf
